test/unit/machine-learning/machine-learning-api-client.spec.ts

```diff
@@ -19,8 +19,8 @@
 import * as _ from 'lodash';
 import * as chai from 'chai';
 import * as sinon from 'sinon';
-import { MachineLearningApiClient, ModelContent,
-  ListModelsOptions } from '../../../src/machine-learning/machine-learning-api-client';
+import { MachineLearningApiClient, ListModelsOptions,
```

Comment:
Are we no longer using ModelContent type in the implementation?

Reply:
It was renamed to ModelOptions

Reply:
It seems earlier these tests were using `ModelContent` in places where they should've used `ModelOptions`. In any case, this look ok.